src/views/Dashboard.vue

```diff
@@ -0,0 +1,249 @@
+<!--
+	- @copyright Copyright (c) 2020 Julius Härtl <jus@bitgrid.net>
+	-
+	- @author Julius Härtl <jus@bitgrid.net>
+	-
+	- @license GNU AGPL version 3 or any later version
+	-
+	- This program is free software: you can redistribute it and/or modify
+	- it under the terms of the GNU Affero General Public License as
+	- published by the Free Software Foundation, either version 3 of the
+	- License, or (at your option) any later version.
+	-
+	- This program is distributed in the hope that it will be useful,
+	- but WITHOUT ANY WARRANTY; without even the implied warranty of
+	- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	- GNU Affero General Public License for more details.
+	-
+	- You should have received a copy of the GNU Affero General Public License
+	- along with this program. If not, see <http://www.gnu.org/licenses/>.
+	-
+	-->
+
+<template>
+	<div v-if="events === null">
+		<div
+			v-for="index in 5"
+			:key="index"
+			class="placeholder-line">
+			<div class="placeholder-icon" />
+			<div class="placeholder-text" />
+		</div>
+	</div>
+	<EmptyContent v-else-if="events.length === 0">
+		<img alt="" :src="imagePath">
+		{{ t('calendar', 'Nothing upcoming for now') }}
+		<template #desc>
+			<p />
+			<button>{{ t('calendar', 'Create a new event') }}</button>
+		</template>
+	</EmptyContent>
+	<div v-else>
+		<ul>
+			<li v-for="event in events" :key="event.id">
+				<a :href="event.deepLink" class="event">
+					<div class="event__title">
```

Comment:
@georgehrke if you’re not using the component, this should be a h3 as well like in the component. :) Then you also don’t need to use the additional `strong` tags for the title.

Reply:
There is a component for it? :) 

I just added the "fetching events part" to Julius' PR.

Reply:
Yep, the component is at https://github.com/nextcloud/nextcloud-vue-dashboard, the other widgets use it already.
We will only have to decide how we display the calendar color indicator (small circle, big circle, etc) and possibly adjust the component.

Also since you use custom structure, please make sure it’s keyboard-focusable (via Tab and Shift-Tab) and then accessible via pressing enter.

Reply:
Let me migrate this code to https://github.com/nextcloud/nextcloud-vue-dashboard in that case :)